ipydatagrid/datagrid.py

```diff
@@ -288,7 +290,7 @@ def __init__(self, dataframe, **kwargs):
     def __handle_custom_msg(self, _, content, buffers):  # noqa: U101,U100
         if content["event_type"] == "cell-changed":
             row = content["row"]
-            column = self._column_index_to_name(content["column_index"])
+            column = DataGrid._column_index_to_name(self._data, content["column_index"])
```

Comment:
any reason we're not using just self here?

Reply:
It's a static method now

Reply:
you can and should call a static method via self

Reply:
👍